LeetCodeSolutions/Program.cs

```diff
@@ -1,3 +1,59 @@
-﻿// See https://aka.ms/new-console-template for more information
+﻿namespace LeetCodeSolutions;
+public class Program
+{
+    public int toRoman(char letter)
+    {
+        switch (letter)
```

Comment:
Would a dictionary use here be more suitable?

Reply:
> Would a dictionary use here be more suitable?

Yes it 100% would. I'll use a Dictionary and request a new review. 
Thank you for the suggestion.